docs/source/guides/data.rst

```diff
@@ -0,0 +1,548 @@
+
+.. testsetup:: *
+
+    from pytorch_lightning.core.Lightning import LightningModule
+    from torch.utils.data import IterableDataSet
+    from pytorch_lightning.trainer.trainer.Trainer import Trainer
+
+.. _data:
+
+#############
+Managing Data
+#############
+
+Continue reading to learn about:
+
+* `Data Objects in Lightning <Data Objects in Lightning_>`_
```

Comment:
I personally find those titles a bit confusing.

Reply:
Can you explain what is confusing? the text or format?

Reply:
`Data Objects` sounds a bit weird to me. Nothing major